From d7887ef223db3be613f5be79ce1ead47ba7b6399 Mon Sep 17 00:00:00 2001
From: Liu Xiaowen <b37945@freescale.com>
Date: Thu, 7 Nov 2013 10:13:51 +0800
Subject: [PATCH 15/45] ENGR00215174 update to pass build for android 4.4.

modify the class ProcessStats import package which is
changed in android 4.4.

Signed-off-by: Liu Xiaowen <b37945@freescale.com>
---
 packages/SystemUI/src/com/android/systemui/LoadSystemTime.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/LoadSystemTime.java b/packages/SystemUI/src/com/android/systemui/LoadSystemTime.java
index eb75849..37f3ec81 100644
--- a/packages/SystemUI/src/com/android/systemui/LoadSystemTime.java
+++ b/packages/SystemUI/src/com/android/systemui/LoadSystemTime.java
@@ -16,7 +16,7 @@
 
 package com.android.systemui;
 
-import com.android.internal.os.ProcessStats;
+import com.android.internal.app.ProcessStats;
 
 import android.app.Service;
 import android.content.Context;
-- 
1.8.0

